practice/src/assignment1/Stack.kt

```diff
@@ -0,0 +1,22 @@
+package assignment1
+
+class Stack<T> {
+
+    private var stack = ArrayList<T>()
+
+    fun push(element: T) {
+        stack.add(element)
+    }
+    fun pop(): T? {
+        if(stack.isEmpty()) {
+            return null
+        }
+        return stack.removeAt(stack.size - 1)
+    }
+    fun size(): Int {
+        return stack.size;
+    }
+    fun find(element: T): Boolean {
+        return stack.contains(element)
+    }
```

Comment:
```suggestion
    fun size() = stack.size
    fun find(element: T) = stack.contains(element)
```

- 불필요한 세미콜론은 없애도 될 것 같습니다
- 일반적으로 size나 contains는 개발자분들이 다 그 의미를 통용하여 사용하기에 별도의 타입을 작성할 필요는 없을 것 같고 그냥 = return 사용하셔도 좋을 것 같습니다.